### ISSUE TITLE
NRK [Errno 11001] getaddrinfo failed

### ISSUE THREAD

**Issue by Mssol94** [rg3/youtube-dl#16324]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.04.25*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.04.25**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '--prefer-ffmpeg', '--all-sub', '--write-thumbnail', '--output', '\\\\10.0.0.18\\j\\Media\\TV-Serier\\Home Ground (2018)\\Season 0%(season_number)s\\Home Ground (2011) - S%(season_number)sE%(episode_number)s - Episode %(episode_number)s - WEBDL-%(height)sp.%(ext)s', '--convert-subs', 'vtt', '--embed-subs', 'https://tv.nrk.no/serie/heimebane/MDRI20001015/sesong-1/episode-10']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2018.04.25
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.16299
[debug] exe versions: ffmpeg 3.4, ffprobe 3.4
[debug] Proxy map: {}
[debug] Using fake IP 84.215.60.232 (NO) as X-Forwarded-For.
[NRKTV] MDRI20001015: Downloading mediaelement JSON
ERROR: Unable to download JSON metadata: <urlopen error [Errno 11001] getaddrinfo failed> (caused by URLError(gaierror(11001, 'getaddrinfo failed'),))
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpad1hmn8r\build\youtube_dl\extractor\common.py", line 519, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpad1hmn8r\build\youtube_dl\YoutubeDL.py", line 2203, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 464, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 482, in _open
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpad1hmn8r\build\youtube_dl\utils.py", line 945, in http_open
  File "C:\Python\Python34\lib\urllib\request.py", line 1185, in do_open
```

---

### Description of your *issue*, suggested solution and other information

I'm getting [Errno 11001] getaddrinfo failed when i try to download from tv.nrk.no. I have also tried other url's than the one provided in the log.

**Comment by kristomu** [comment]
This might be the same issue as 13796, but in reverse. It appears that NRK has changed their API domain name back from ``psapi-ne.nrk.no`` to ``psapi-we.nrk.no`` (as a simple nslookup of these two names can confirm).

**Comment by surfmuggle** [comment]
I was able to fix the issue  #25670  but do not know which change did the fix

In my router i changed the dns server and turned on ipv4 only
In cmd.exe i entered
 - ipconfig /all
 - ipconfig /release
 - ipconfig /renew

After all this i was able to download again.
